src/ceph-disk

```diff
@@ -1373,13 +1958,26 @@ def prepare_journal_dev(
                     ),
                 '--mbrtogpt',
                 '--',
-                journal,
+                journal_dev,
                 ]
             )
 
         update_partition(journal, 'prepared')
 
+        # wait for udev event queue to clear
+        udev_settle()
+
         LOG.debug('Journal is GPT partition %s', journal_symlink)
+        #
```

Comment:
the logic below should not be necessary: I don't think there is a need to wait for the journal device to show up: it was created by sgdisk a few lines above and we already know which device name it will have. Or do you have another problem in mind ?